joomla/joomla-cms
# JoomlaTinyMCE.setupEditors() overwrites element style

### Summary of Changes
Fix for JoomlaTinyMCE.setupEditors() overwriting element style(width & height) with default values


### Testing Instructions
Create a page with element:

```
<div class="js-editor-tinymce"><textarea style="width: 1234px; height: 345px;"></textarea></div>
```
Make sure tinymce.min.js is included inside the head:
```
<script src="/<joomla root>/media/editors/tinymce/js/tinymce.min.js?165d29f84be303af93c70376b473e0fb" type="text/javascript"></script>
```
### Expected result
Inline element style is used instead of default values


### Actual result
Default values are used


### Documentation Changes Required
